apps/communications/contacts/style/app.css

```diff
@@ -513,6 +514,7 @@ li button.icon-find-and-merge:before {
 
 nav[data-type="scrollbar"] {
   z-index: 6;
+  display: none;
```

Comment:
If this is being hidden for non-touch, consider just removing it completely instead.